.idea/deployment.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="PublishConfigData" remoteFilesAllowedToDisappearOnAutoupload="false">
+    <serverData>
+      <paths name="test">
+        <serverdata>

```

Comment:
@ReginaBorisova 
Seems that files in .idea folder can be added to gitignore and removed from the repo.
use `git rm --cached path_to_file` command to remove file from git but keep local copy (create a backup in case you will remove files)

Reply:
Done together